### PR TITLE
Adding commit logging and T/F return on pattern_writer

### PR DESCRIPTION
**Pull request by dosumis**

### CODE REVIEW

**Comment by dosumis** [comment]
Commit logging on pattern_writer + tests.

File: src/uk/ac/ebi/vfb/neo4j/KB_tools.py [hunk 1]
```diff
@@ -416,7 +422,9 @@ def test_edge_addition(self):
         missed_edges = [x['match_count'] for x in dc if x and (0 in x['match_count'].values())]
         if missed_edges:
             for e in missed_edges:
-                warnings.warn("No match found for %s" % str([k for k, v in e.items() if not v]))
+                m = "No match found for %s" % str([k for k, v in e.items() if not v])
```
Comment (matentzn) [inline src/uk/ac/ebi/vfb/neo4j/KB_tools.py]: how many could these be in the worst case?

Comment (dosumis) [reply src/uk/ac/ebi/vfb/neo4j/KB_tools.py]: I don't expect to hit these errors.  They will be caught be pre-commit checks unless the DB has malformed property nodes.

Comment (matentzn) [reply src/uk/ac/ebi/vfb/neo4j/KB_tools.py]: No problem then!

**Pull request merged**